src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -45,19 +56,60 @@ export class QuerySummary extends Component {
    */
   static options: IQuerySummaryOptions = {
     /**
-     * Specifies whether to display the search tips to the end user when there are no search results.
+     * Specifies whether to hide the information about the currently displayed range of results and only display the
+     * search tips instead.
+     *
+     * Default value is `false`.
+     */
+    onlyDisplaySearchTips: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+
+    // TODO : Documentation review
+    /**
+     * Specifies whether to display the a message to the end user when there are no search results.
      *
      * Default value is `true`.
      */
-    enableSearchTips: ComponentOptions.buildBooleanOption({ defaultValue: true }),
+    enableNoResultsFoundMessage: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
+    // TODO : Documentation review : I'm not sure if we should put an example in the description and if the description is precise enough?
     /**
-     * Specifies whether to hide the information about the currently displayed range of results and only display the
-     * search tips instead.
+     * Specifies a custom message to display when there are no search results.
      *
-     * Default value is `false`.
+     * A query tag ({@link QUERY_TAG}) can be added in the message. This tag will
+     * be replaced by the search box input who triggered the no results found page.
+     *
+     * Ex.:
+     *
+     * Searchbox input : "ThereIsNoResults"
+     *
+     * {@link QUERY_TAG} = <%-query%>
```

Comment:
I would use `${query}` instead of `<%- query %>`, just for uniformity reasons, since we have a similar concept in `ResultLink` : 

https://coveo.github.io/search-ui/components/resultlink.html#options.titletemplate

Reply:
Good point! I changed it!